internal/services/datadog/azurerm_datadog_monitor_sso_configuration.go

```diff
@@ -63,12 +64,10 @@ func resourceDatadogSingleSignOnConfigurations() *pluginsdk.Resource {
 				ValidateFunc: validate.DatadogEnterpriseApplicationID,
 			},
 
-			"single_sign_on_enabled": {
+			"single_sign_on": {
 				Type:     pluginsdk.TypeString,
 				Required: true,
 				ValidateFunc: validation.StringInSlice([]string{
-					// @tombuildsstuff: other options are available, but the Create handles this as a boolean for now
-					// should the field be a boolean? one to consider for 4.0 when this resource is inlined
 					string(singlesignon.SingleSignOnStatesEnable),
 					string(singlesignon.SingleSignOnStatesDisable),
```

Comment:
Can you add support for the other values here as well? From what we saw, I doubt we'll need them but we may as well let people use them if they want